regression/tests/lms/test_demo.py

```diff
@@ -26,3 +26,5 @@ def test_login(self):
         """
         self.login_page.visit()
         self.login_page.login(self.DEMO_COURSE_USER, self.DEMO_COURSE_PASSWORD)
+        self.assertEquals(self.login_page.q(css='.wrapper-header-courses .header-courses').text[0].lower(),
```

Comment:
@benpatterson I been thinking, should we remove this file? It was intended for the testing purposes.